10.1/sass/style.scss

```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+}
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  
+}
+
+.carousel-cell {
+  width: 100%;
+  min-height: 250px;
+  height: 50vh;
+  background-position: center 40%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 2rem;
+  text-align: center;
+  color: #fff;
+  text-shadow: 2px 2px #333;
+}
+.flickity-page-dots {
+  display: none;
+}
```

Comment:
to zalatw za pomoca opcji przekazywanej w obiekcie konfiguracyjnym przy tworzeniu karuzeli Flickity